src/tim/vx/graph.cc

```diff
@@ -185,7 +220,11 @@ bool GraphImpl::Setup() {
 
 bool GraphImpl::Compile() {
   bool status = true;
-
+  if (not_consumed_input_cnt_ > 0 || not_consumed_output_cnt_ != 0) {
```

Comment:
why not use "not_consumed_output_cnt >0", do you mean it may be negative?

Reply:
a input tensor can be bind repeatly,may be negative